core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -364,6 +365,45 @@ public ServerBuilder port(ServerPort port) {
         return this;
     }
 
+    /**
+     * Adds the specified {@link ServerPort} to loopback {@code localAddress}.
+     *
+     * @see <a href="#no_port_specified">What happens if no HTTP(S) port is specified?</a>
+     */
+    public ServerBuilder loopbackPort(int port) {
+        return port(new InetSocketAddress(NetUtil.LOCALHOST, port));
+    }
```

Comment:
How about removing this method in favor of `loopbackPort(int port, SessionProtocol... protocols)`?